src/main/java/org/jenkinsci/plugins/ghprb/GhprbCause.java

```diff
@@ -53,7 +53,12 @@ public String getCommit() {
 	}
 	
 	public boolean isMerged() {
-		return merged;
+        // hardcode hack: never automatically merge a commit for a pull request build.
+        // see various bugs that are affected by this. Header descriptions for edx are in
+        // https://openedx.atlassian.net/browse/TE-703
+
+        //return merged;
+        return false;
```

Comment:
Does it make more sense to instead change the object's merged attribute? I.e. where
```
this.merged = merged;
```
That way the getShortDescription function will also return the right thing?